gap/almostsimple.gi

```diff
@@ -1011,6 +1025,12 @@ end;
 
 # Looking at element orders to determine which sporadic it could be:
 
+# RECOG.SporadicsElementOrders contains the element orders that apear in the
+# corresponding sporadic group in RECOG.SporadicsNames each with the
+# probability in RECOG.SporadicsProbabilities
+# RECOG.SporadicsSizes contains the sizes of the sporadic groups in
+# the order of RECOG.SporadicsNames
+# RECOG.SporadicsKillers contain orders that apear relatively often
```

Comment:
.This paragraph lacks punctuations `,` and `.`, which IMHO makes it hard to read.